Src/DependencyCollector/Shared/Implementation/SqlClientDiagnostics/SqlClientDiagnosticSourceListener.cs

```diff
@@ -2,8 +2,6 @@ namespace Microsoft.ApplicationInsights.DependencyCollector.Implementation.SqlCl
 {
     using System;
     using System.Collections.Generic;
```

Comment:
removing this import should have caught the issue earlier. Now we can confirm that we rely purely on reflection via PropertyFetchers.